packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -121,6 +121,38 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       buckets: [0.1, 1, 2, 4, 10],
       labelNames: ["source"],
     }),
+    executionBlockProductionTimeSteps: register.histogram<"step">({
+      name: "beacon_block_production_execution_steps_seconds",
+      help: "Detailed steps runtime of execution block production",
+      buckets: [0.1, 1, 2, 4, 10],
```

Comment:
buckets value seems to big for each step. If a step takes more than 1s then it's already an issue so I suggest removing `2, 4, 10` and track more `< 1` buckets

Reply:
I removed the buckets completely, as these are not intended in the metric graph we need.